src/pages/content/index.vue

```diff
@@ -1,72 +1,109 @@
 <template>
-	<section class="section">
-		<searchbar
-			v-model.lazy="searchQuery"
-			class="searchbar"
-			placeholder="Suche nach..."
-		/>
-		<pagination v-model="skippedItems" :state="pagination" />
-		<div class="columns">
-			<div v-for="content of searchResults" :key="content._id" class="column">
-				<pre>
-					{{ content }}
-				</pre
-				>
+	<section>
+		<div v-if="scrollY > 115" class="content__back-to-top">
+			<p @click="$_backToTop">
+				<i class="icon fa fa-chevron-circle-up"></i>
+			</p>
+		</div>
+		<div class="content">
+			<searchbar
+				v-model.lazy="searchQuery"
+				class="content__searchbar"
+				placeholder="Suche nach..."
```

Comment:
- [ ] Please use the language files here as well
```suggestion
				:placeholder="$t('pages.content.index.search_for')"
```